cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -149,7 +152,12 @@ public void initialize(MasterEnvironmentContext context) throws IOException, Api
                namespace, podKillerSelector, delayMillis);
     }
 
-    twillRunner = new KubeTwillRunnerService(namespace, discoveryService, podInfo, resourcePrefix,
+    Map<String, Float> kubeScalingFactors = new HashMap<String, Float>();
+    kubeScalingFactors.put("cpu", Float.parseFloat(conf.getOrDefault(CPU_SCALING_FACTOR, DEFAULT_SCALING_FACTOR)));
```

Comment:
Why need to copy the config? You can pass the whole `conf` object to the runner service. In future there could be other configurations being added.

Reply:
Done.